crypto/hd/algo.go

```diff
@@ -69,3 +73,25 @@ func (s secp256k1Algo) Generate() GenerateFn {
 		return &secp256k1.PrivKey{Key: bzArr}
 	}
 }
+
+type sr25519Algo struct {
+}
+
+func (s sr25519Algo) Name() PubKeyType {
+	return Sr25519Type
```

Comment:
Wheere is this defined?

Reply:
nvm Found it here: https://github.com/psaradev/cosmos-sdk/blob/crypto/keys/sr25519/crypto/hd/algo.go#L24